packages/arb-token-bridge-ui/src/hooks/useERC20L1Address.ts

```diff
@@ -36,4 +56,4 @@ const useERC20L1Address = ({
   return { data, isLoading: isValidating }
 }
 
-export { useERC20L1Address }
+export { getL1ERC20Address, useERC20L1Address }
```

Comment:
I find it odd that this is exported and used in `useArbTokenBridge`

if it's used in multiple files, shouldn't it be in a separate file rather than in one of the files it's used in?

Reply:
to be more specific, I was speaking of `getL1ERC20Address`
in all 3 files those functions (not the hooks) are not used in other files, and except in `useWithdrawals`, the functions were not exported